page-sidebar.php

```diff
@@ -18,49 +18,22 @@
 
 
 get_header(); ?>
+<?php get_template_part('inc/sub-headerException'); ?>
+		
+		<div id="stage" class="inner" role="main">
 
-		<?php get_template_part('inc/breadcrumbs', 'noChild'); ?>
+			<?php while ( have_posts() ) : the_post(); ?>
+
+				<div id="content" class="content has-sidebar">
+
+				<?php get_template_part( 'inc/content', 'page' ); ?>
```

Comment:
I see you grabbing the content of the page here, but i don't see the title (which I think we should have somewhere - maybe I missed it) or other stuff (which perhaps we don't need here). Maybe we can deploy the branch to dev and I can see it in action...